app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -130,9 +130,9 @@ private fun Content(
         onNext = { focusManager.moveFocus(FocusDirection.Down) }
     )
     if (firstName.isNotEmpty() && !stateFirstName.error.isValid()) {
-        if (stateFirstName.error == LoginError.NameError.Length) {
+        if (stateFirstName.error == com.hamdan.forzenbook.core.LoginError.NameError.Length) {
```

Comment:
Import this so you don't have these giant package names on classes

Reply:
yea, I believe I am in all cases, but it auto put that ahead on everything, like all spots like this are greyed out.